docs/source/orquesta/operations.rst

```diff
@@ -27,7 +27,22 @@ are no more active tasks.
 Re-running Workflow Execution
 -----------------------------
 
-A workflow execution can be re-run on error. In the current version, the workflow execution can only
-be re-run from the beginning. Re-running the workflow execution from the beginning is exactly like
-re-running any |st2| execution with the command ``st2 execution re-run <execution-id>``. The re-run
-is a completely separate execution with a new execution ID in |st2|.
+A workflow execution can be re-run from the beginning. It is exactly like re-running any |st2|
+execution with the command ``st2 execution re-run <execution-id>``. The re-run is a completely
+separate action execution with a new action execution ID in |st2| and a new workflow execution
+is created for the new action execution.
+
+Re-running Workflow Execution from Task(s)
+------------------------------------------
+
+There are use cases where users want to re-run the workflow execution for certain task(s). Orquesta
+allows workflow execution to be re-run from any task(s) as long as the workflow execution is in a
+completed state (succeeded, failed, or canceled). Let's say we have a sequential workflow execution
+that executes task1 -> task2 -> task3. The workflow execution can be re-run from any task with
+the command ``st2 execution re-run <execution-id> --tasks <task_name>``. A new action execution is
```

Comment:
The `--tasks` argument is plural - does that mean I can specify multiple tasks to rerun?

Example workflow:
`task_1` -> `task_2` -> `task_3`

Do I need to specify `st2 execution re-run <execution-id> --tasks task_1 task_2 task_3`?

Or will `st2 execution re-run <execution-id> --tasks task_1` automatically continue to run `task_2` and `task_3`?

Reply:
@blag I updated the docs section with example on passing multiple tasks.  In your example here, you specified the task(s) where you want to start rerunning.  In this case, you should only pass task1.  This reminds me I need to add a fix/unit tests for you example.  Even in the case where users pass task1, task2, and task3.  The workflow should only run from task1.  Good catch.